package.json

```diff
@@ -144,8 +144,8 @@
     "typescript": "^4.3.5"
   },
   "dependencies": {
-    "@chakra-ui/react": "^1.6.9",
-    "@d-i-t-a/reader": "^2.0.0-beta.6",
+    "@chakra-ui/react": "1.6.9",
+    "@d-i-t-a/reader": "2.0.0-beta.6",
```

Comment:
Keeping these two fixed versions because the new chakraui version is causing a build error on the Text component and version >2.0.0-beta.6 causes parcel build error...